katello-agent/katello-agent.spec

```diff
@@ -33,6 +33,11 @@ Requires: yum-plugin-security
 Requires: yum-security
 %endif
 
+%if 0%{?fedora} > 18 || 0%{?rhel} > 6
+Requires: python2-tracer >= 1.6.12 
+Requires: python-rhsm
+%endif
```

Comment:
curious if you had thought about making this a subpackage that is only built on those platforms?